lib/farm_ng/oak/oak_client.py

```diff
@@ -0,0 +1,76 @@
+import logging
```

Comment:
Note, do we need a separate directory for lib which is intended only for circuitpython versus desktop? I think this now conflicts with the instructions for using the farm_ng.adk for the microcontroller kit examples where we instruct the user to drag and drop this folder onto the feather (which will quickly run out of space and memory). @Hackerman342 ideas?

Reply:
I think we need this to work well with basic filesystem and command line tools before we can support a vs code environment.

Suggest having two lib folders, on that's for circuitpython and one that's for code that should run on a computer as a simple starting point.

Reply:
sounds good to me